compendium/LogService/include/cppmicroservices/logservice/Logger.hpp

```diff
@@ -0,0 +1,121 @@
+#ifndef CPPMICROSERVICES_LOGGER_H_
+#define CPPMICROSERVICES_LOGGER_H_
+
+#include "cppmicroservices/ServiceReferenceBase.h"
+
+#include <cstdint>
+#include <exception>
+#include <string>
+#include <functional>
+
+namespace sinks
+{
+    class sink;
+}
+
+namespace spdlog
+{
+    class logger;
+    using sink_ptr = std::shared_ptr<sinks::sink>;
+} // namespace spdlog
+
+namespace cppmicroservices::logservice
+{
+        /** @}*/
+
+        /**
+         * \ingroup MicroService
+         * \ingroup gr_logservice
+         *
+         * Provides methods for bundles to write messages to the log.
+         * Logger interface defines several methods for each of the defined LogLevels.
+         *
+         */
+
+	enum class LogLevel
+        {
```

Comment:
Enums are typically denoted by uppercase letters, maybe should be here as well

Reply:
I removed this altogether since we are not using this as of now in our code. Maybe in future, when we take LoggerContext (An interface that allows the configuration of effective logging levels for a Bundle) into picture, we can add this.